app/page.tsx

```diff
@@ -1,11 +1,15 @@
 import AuthButton from '../components/AuthButton';
+import { Button } from '@/components/ui/button';
```

Comment:
Remove changes from this file unless it's breaking from the changes in the button component. This work will be completed when we touch the main page.

Reply:
Done. Commited and will be pushed when all other changes are complete.